docs/luigi-ux-features.md

```diff
@@ -103,11 +103,11 @@ To show a loading indicator before Luigi Core or your first micro frontend is re
 
 ```html
 <div luigi-app-loading-indicator>
-  <div class="fd-busy-indicator--m" aria-hidden="false" aria-label="Loading">
-    <div class="fd-busy-indicator--circle-0"></div>
-    <div class="fd-busy-indicator--circle-1"></div>
-    <div class="fd-busy-indicator--circle-2"></div>
-  </div>
+	<div class="fd-busy-indicator fd-busy-indicator--l" aria-hidden="false" aria-label="Loading">
```

Comment:
```suggestion
	<div class="fd-busy-indicator fd-busy-indicator--l" aria-hidden="false" aria-label="Loading">
```
Since all other instances are `m` instead of `l`, it would be good to make it consistent.